### ISSUE TITLE
about IE9-IE11

### ISSUE THREAD

**Issue by unclemeric** [vuejs-templates/webpack#1244]
Is the latest version is not support ie9-ie11？i can't run success on IE

**Comment by LinusBorg** [comment]
Ît is as much (or little) supported as previous versions:

* Vue itself supports >=IE9
* But to use some ES6 Javascript features (like Promises) you need to include polyfills.

What problems to you have when trying to use IE? Please be more specific.

**Comment by unclemeric** [comment]
yes,i know that Vue supports >= IE9.
i had install babel-polufill，
```
entry: {
    app: ['babel-polyfill','./src/main.js']
  },
```
but it's no use.
before version 2.9.2(the version of use express to run start) can run successful.
error message:
<img width="340" alt="6820364e-688b-4a99-96e0-ed1181a185ee" src="https://user-images.githubusercontent.com/6083914/34868095-a700c2ec-f7bd-11e7-9f77-0bb83ecb9eb7.png">

**Comment by LinusBorg** [comment]
I don't speak chinese, I can't read this :(

**Comment by Hornet-Wing** [comment]
I had a similar issue running a clean install of this template in IE11. The source of the bug is webpack-dev-server 2.10.x.

Downgrading to 2.9.6 fixed the issue for now:
```
npm i webpack-dev-server@2.9.6 -D
```

It looks like its being looked at and fixed here:
https://github.com/webpack/webpack-dev-server/issues/1268

**Comment by unclemeric** [comment]
@LinusBorg @Hornet-Wing thank you very much!

**Comment by unclemeric** [comment]
@LinusBorg @Hornet-Wing thank you very much!

**Comment by LinusBorg** [comment]
upstream issue was fixed with WDS 2.11 ([Release log](https://github.com/webpack/webpack-dev-server/releases/tag/v2.11.0))

**Comment by bangStweart** [comment]
![image](https://user-images.githubusercontent.com/41061365/51960890-6bdac980-248d-11e9-8f5c-35ff325ed46e.png)

Btw, I have got this error on ie11.

![image](https://user-images.githubusercontent.com/41061365/51960913-7dbc6c80-248d-11e9-9175-38344cb298ec.png)
